src/components/Filter/scss/Filter.scss

```diff
@@ -0,0 +1,159 @@
+        flex-grow: 5;
+        position: relative;
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+      }
+
+        .list-filter__item__container__input__appendage {
+          width: 1.2rem;
+          height: 1.2rem;
+          position: absolute;
+          right: 2.5rem;
+          top: 50%;
+          transform: translateY(-50%);
+          line-height: 1;
+          & > .btn--text {
+            svg {
+              width: 1.2rem;
+              height: 1.2rem;
+            }
+            &:hover {
+              svg {
+                fill: $clr-base-dk;
+              }
+            }
+          }
+        }
+
+        .list-filter__item__container__input__suffix {
+          width: $list-filter-search-size;
+          height: $list-filter-search-size;
+          display: block;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          & > i,
+          & svg {
+            display: block;
+            width: $list-filter-search-size;
+            height: $list-filter-search-size;
+          }
+        }
+
+    .list-filter__item--predefined {
+      display: flex;
+      flex-direction: column;
+      position: relative;
+      align-self: flex-end;
+      &.is-open {
+        & .list-filter__item__filter-collapsable {
+          max-height: 50rem;
+          opacity: 1;
+          overflow: visible;
+        }
+      }
+    }
+      .list-filter__item__filter-btn-container {
+        width: 100%;
+      }
+        .list-filter__item__filter-btn {
+          padding: 1.3em;
+          min-width: auto;
+          width: 1em;
+          height: 2.7em; // bad - magic number to fit input height
+          border-radius: 0;
+          svg {
+            position: absolute;
+            left: 50%;
+            top: 50%;
+            transform: translateX(-50%) translateY(-50%);
+            width: 2rem;
+            height: 2rem;
+          }
+        }
+      .list-filter__item__filter-collapsable {
+        max-height: 0;
+        opacity: 0;
+        overflow: hidden;
+        position: absolute;
+        top: 105%;
+        z-index: 9;
+        background-color: $clr-ntrl-min;
+        box-shadow: 0px 0px 10px $clr-ntrl-max--transp-ltr;
+        transition: all 350ms ease-in-out;
+        &:before {
+          content: '';
+          border-bottom: 10px solid $clr-ntrl-min;
+          border-left: 10px solid transparent;
+          border-right: 10px solid transparent;
+          height: 0;
+          left: 7%;
+          position: absolute;
+          top: -10px;
+          transform: translateX(-50%);
+          width: 0;
+        }
+      }
+        .list-filter__item__filter-collapsable__item {
+          border-bottom: 1px solid $clr-base-lt;
+        }
+          .list-filter__item__filter-collapsable__item__btn {
+            @include soft--smaller--sides;
+            @include soft--smallest--ends;
+          }
```

Comment:
not sure about all of this... can't request changes though because I'm not sure what to recommend or why I'm unsure about it. leaving this as a footnote.

Reply:
agreed, there are simply soo many opinions here - will have to greatly simplify